go.mod

```diff
@@ -58,7 +58,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.11.27
 	github.com/Azure/go-autorest/autorest/adal v0.9.20
 	github.com/Azure/go-autorest/autorest/to v0.4.1-0.20210111195520-9fc88b15294e
-	github.com/AzureAD/microsoft-authentication-library-for-go v0.5.1
+	github.com/AzureAD/microsoft-authentication-library-for-go v0.7.0
```

Comment:
Hey Markus, do you know why we have such a significant version bump for this package and the other?

Reply:
Not sure, but initially I had thought it was with some of the import changes. But once I removed the import entries, this version bump still stayed. We could always try to push it back to 0.5.1 and I run a quick test?

Reply:
I think that would be great, I would rather dev be in charge of version bumps instead of us doing it unintentionally

Reply:
I agree with that. That'd be really great to discard all `go.mod` and `go.sum` changes and try it, as these changes might affect other things in the rancher.

Reply:
Tested locally and it was fine. Unsure why latest commit doesn't show those changes, will try again.